docs/inferences/ocr.md

```diff
@@ -43,4 +43,6 @@ if "Landing" in frame.predictions:
     print("Found 'Landing' written in the text!")
 ```
 
-The results may vary depending on the image quality, the font, and the language. Try with your own images to see how well the model performs, and [provide us feedback about the results](https://github.com/landing-ai/landingai-python/issues/new).
\ No newline at end of file
+The results may vary depending on the image quality, the font, and the language. Try with your own images to see how well the model performs, and [provide us feedback about the results](https://github.com/landing-ai/landingai-python/issues/new).
+
+The `OcrPredictor` class accepts a couple of extra parameters that can help you finetune the results you get. Please check its documentation for more details.
```

Comment:
Instead of `Please check its documentation for more details.`, can we add a paragraph here explaining which are the options and its values?

Reply:
We could, but isn't that the job of the generated documentaion? I mean, I wouldn't like to write every specific change in parameter here and in the docstring as well. I assumed it was the expected thing because you could se that `mode` and `regions_of_interest` were not documented here, but only in the docstring.

Reply:
We need to explain what those parameters are, which are the benefits of fine tuning them and how they impact the prediction results. This seems a bit too much for just the autogenerated docstring.

Not having `mode` and `regions_of_interest` documented here is a mistake that should be fixed too, otherwise it is **very** unlikely that people will discover those parameters by themselves.